src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -73,18 +84,18 @@ To avoid bundling the script, you can use the `is:inline` directive.
 ```
 
 :::note
-Astro's default bundling behavior will be disabled in some situations. In particular, adding `type="module"` or any attribute other than `src` to a `<script>` tag will cause Astro to treat the tag as if it had an `is:inline` directive. The same will be true when the script is written in a JSX expression.
+Astro will not process your script tags in some situations. In particular, adding `type="module"` or any attribute other than `src` to a `<script>` tag will cause Astro to treat the tag as if it had an `is:inline` directive. The same will be true when the script is written in a JSX expression.
 :::
 
 📚 See our [directives reference](/en/reference/directives-reference/#script--style-directives) page for more information about the directives available on `<script>` tags.
 
-### Script loading
+### Include javascript files on your page
 
 You may want to write your scripts as separate `.js`/`.ts` files or need to reference an external script on another server. You can do this by referencing these in a `<script>` tag’s `src` attribute.
 
 #### Import local scripts
 
-**When to use this:** If your script lives inside of `src/`.
+**When to use this:** when your script lives inside of `src/`.
 
 Astro will build, optimize, and add these scripts to the page for you, following its [script bundling rules](#script-bundling).
```

Comment:
```suggestion
Astro will build, optimize, and add these scripts to the page for you, following its [script bundling rules](#how-astro-processes-scripts).
```
This will fix the fact that the heading has been changed, but given that the wording has changed, please also check this sentence to see that it accurately describes what it's linking to. i.e. it claims to be taking you to "script bundling rules" but now takes you to a more generic section. Verify you think this still makes sense as a place to link and/or whether this sentence is still phrased correctly for the new context! 🙌 

(And, if you follow my suggestion to change that heading to something else, actually, that should be reflected here.)

Reply:
Yeah, I think this still applies.